### PR TITLE
13.0 mig sale force invoiced

### PR DESCRIPTION
**Pull request by jvpascual**
Migrated.
we add the feature of being able to pass the invoice to 'fully invoiced' without having to have the sale state in done.

Thanks.

### CODE REVIEW

**Comment by sudhir-erpharbor** [comment]
Code review LGTM

File: sale_force_invoiced/__manifest__.py [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
                'Invoiced',
-    'version': '12.0.1.0.1',
+    'version': '13.0.1.0.0',
     "author": "Eficent,"
               "Odoo Community Association (OCA)",
     'category': 'sale',
     'license': 'AGPL-3',
     'website': "https://github.com/OCA/sale-workflow",
-    'depends': ['sale'],
+    'depends': ['sale', 'account'],
```
Comment (MiquelRForgeFlow) [inline sale_force_invoiced/__manifest__.py:14]: no need to add here `account`, it's already a dependency (of `payment`, which is already a dependency of `sale`).

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: sale_force_invoiced/__manifest__.py (hunk 1 above)
Comment (jvpascual) [reply sale_force_invoiced/__manifest__.py:14]: @mreficent, thanks for the correction, i'll solve it.

File: sale_force_invoiced/README.rst [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+Authors
+~~~~~~~
+
+* Eficent
+
+Contributors
+~~~~~~~~~~~~
+
+* Jordi Ballester <jordi.ballester@eficent.com>
+* Juan Vicente Pascual <jvpascual@puntsistemes.es>
```
Comment (kongrattapong) [inline sale_force_invoiced/README.rst]:
```suggestion
* 
```
Generate README following  https://github.com/OCA/maintainer-tools/blob/master/README.md

**Review by kongrattapong — CHANGES_REQUESTED** [review]
@jvpascual

**Comment by ph-obs** [comment]
Hey guys. Will this get merged any time soon?
@jvpascual

**Comment by AaronHForgeFlow** [comment]
as soon as comments attended I guess

**Comment by jvpascual** [comment]
Comments attended.

**Comment by AaronHForgeFlow** [comment]
still pending comment from  @MiquelRForgeFlow  and CI is red.

**Comment by jvpascual** [comment]
i'll check it.

**Review by HaraldPanten — APPROVED** [review]
Functional review OK 👍in local environment . But you should check comments @jvpascual 

Thank you!

**Review by danimv5 — APPROVED** [review]
Functional review! LGTM

**Review by eduardgm — APPROVED** [review]
Functionally checked, LGTM 👍🏻

**Comment by AaronHForgeFlow** [comment]
The module is already migrated. You can close this PR.

**Pull request closed without merging**